shared/searchv3/user-input/index.js.flow

```diff
@@ -23,4 +23,6 @@ export type Props = {
   onMoveSelectDown: () => void,
 }
 
-export default class UserInput extends Component<void, Props, void> {}
+export default class UserInput extends Component<void, Props, void> {
+  focus: () => void
```

Comment:
Is this necessary? isn't it just a local thing that outside users don't need to know about?